### PR TITLE
chore: bump packages for mobx

### PR DESCRIPTION
**Pull request by jonezque**
https://www.notion.so/eb96722b397d42c9908002ca06af88e3?v=35dcf02f76284d8e8efe0b67f7a04234&p=4045c924e1f04481823820ba6ea4a54c

### CODE REVIEW

**Comment by r0x4xn** [comment]
App <strong>website</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://website-p0k9z1slm-prepo.vercel.app'>https://website-p0k9z1slm-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_OtZFmASyChHVcxNEAY2NfOQoYQ56"></hidden>

**Comment by r0x4xn** [comment]
App <strong>react-boilerplate</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://react-boilerplate-ladea2bmf-prepo.vercel.app'>https://react-boilerplate-ladea2bmf-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_3Lydru5gdN5RuqWFQBT8AZrbiROq"></hidden>

**Comment by r0x4xn** [comment]
App <strong>acquisition-royale-landing-page</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://acquisition-royale-landing-page-rd92xhfs4-prepo.vercel.app'>https://acquisition-royale-landing-page-rd92xhfs4-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_So4gfuKdA7SYrdLpc20mSTLyCWAz"></hidden>

**Comment by r0x4xn** [comment]
App <strong>core-dapp</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://core-dapp-fecpbm4i1-prepo.vercel.app'>https://core-dapp-fecpbm4i1-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_JWVJ3WoV71UcOmB52qrcezjyBL3V"></hidden>

File: apps/frontend/website/package.json [hunk 1]
```diff
@@ -21,10 +21,10 @@
...
     "configcat-js-ssr": "3.10.0",
     "d3-ease": "3.0.1",
-    "mobx": "6.5.0",
-    "mobx-react": "7.3.0",
-    "mobx-react-lite": "3.3.0",
-    "mobx-state-tree": "5.1.3",
+    "mobx": "6.6.0",
+    "mobx-react": "7.5.0",
+    "mobx-react-lite": "3.4.0",
+    "mobx-state-tree": "5.1.5",
```
Comment (r0x4xn) [inline apps/frontend/website/package.json:27]: Do we need mobx on this app?

Comment (jonezque) [reply apps/frontend/website/package.json]: good catch, blindlessly updated package without checking

Comment (r0x4xn) [reply apps/frontend/website/package.json:27]: Right, we can fix that on a separate PR. Mobx is not being used on website, we don't really have the need for state management there

**Review by r0x4xn — APPROVED** [review]

**Pull request merged**